waspc/data/Generator/templates/react-app/src/auth/useAuth.ts

```diff
@@ -1,16 +1,19 @@
+import { deserialize as superjsonDeserialize } from 'superjson'
 import { useQuery } from '../queries'
 import api, { handleApiError } from '../api'
 import { HttpMethod } from '../types'
+// todo(filip): turn into a proper import
+import { type SanitizedUser as User } from '../../../server/src/_types/' 
 
-export default function useAuth(queryFnArgs, config) {
+export default function useAuth(queryFnArgs?: unknown, config?: any) {
```

Comment:
It seems like `useQuery` will send `queryFnArgs` to `getMe`, but `getMe` doesn't expect them so they end up ignored.

Todo: make an issue for this or investigate.